crates/crates_io_database/src/schema.rs

```diff
@@ -538,6 +538,9 @@ diesel::table! {
         ///
         /// (Automatically generated by Diesel.)
         token_generated_at -> Nullable<Timestamptz>,
+        /// Whether this email is the primary email address for the user.
+        #[sql_name = "is_primary"]
```

Comment:
`primary` is a reserved keyword in Postgres, so we use `is_primary` as the column name and then alias it to `primary` in the Diesel schema to be consistent with `verified`.